config/samples/install_v1alpha1_lookoutingester.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     app.kubernetes.io/managed-by: kustomize
     app.kubernetes.io/created-by: armada-operator
   name: lookoutingester-sample
-  namespace: armada
+  namespace: default
```

Comment:
can you revert this?  I changed these because apparently GR deploys everything in the armada namespace.

For local kind testing we may need to just create a namespace for armada.

Reply:
i think we just need to instead use armada namespace in he Helm install of postgres

Reply:
These are unrelated.  If you install postgres using helm you still need to install that into the armada namespace.  You would need to make sure the namespace matches for all of these components.